Exiled.API/Features/Player.cs

```diff
@@ -122,6 +122,21 @@ public Player(GameObject gameObject)
         /// </summary>
         ~Player() => HashSetPool<int>.Shared.Return(TargetGhostsHashSet);
 
+        /// <summary>
+        /// Gets all the current players of a specific role.
+        /// </summary>
+        public static Dictionary<RoleType, HashSet<Player>> InstantiatedRolesToPlayers { get; internal set; } = new();
```

Comment:
First, I dont think that's those dicts are really needed.
To me, it's just an inefficient memory usage, LINQ in `Player.Get` better as for me.

Reply:
I do agree storing HashSet<Player> is a little expensive; however, speed wise of access has to be significantly better than LINQ foreach every single time you want a specific player of Role, Side, and Team. However, it's up to the feedback I get to decide if this is worth the time (AKA you, Nao, Build, etc)